test/http.spec.js

```diff
@@ -41,8 +41,8 @@ describe(`http backend using ${hasXMLHttpRequest() ? 'XMLHttpRequest' : 'fetch'}
             errO = err
             dataO = data
             resolve(true)
-            setTimeout(() => reject(new Error('timeout')), 1500)
           })
+          setTimeout(() => reject(new Error('timeout')), 1500).unref()
```

Comment:
I moved this line so that it behaves as a proper timeout check against the backend.read call.